### ISSUE TITLE
no-get-inner-outer-html generates false positives

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#77]
`no-get-inner-outer-html` rule is currently too broad. It would warn about this usage:

```js
obj.getInnerHtml();
```

The problem is, the rule does not check if an object is an element finder.

**Comment by alecxe** [comment]
Fixed by 57092ca598eb8ff32a5817a4135d7b16a4cda91a.
